client/src/style/app.scss

```diff
@@ -19,7 +19,7 @@ h2 {
   text-transform: uppercase;
   letter-spacing: $spacing-xs;
   font-size: 2.5vw;
-  margin-top: -10px;
+  margin-top: 0px;
```

Comment:
Thanks for the PR! Also, why not simply remove the entire `margin-top` line of code? I guess it will have the same effect?

Reply:
Browsers have a `user agent style sheet`. 
It adds default styling to tags. 
Setting margin-top will override the default for h2 here.

Reply:
Thanks, @Kvaibhav01 for reviewing it.